text/expressions_test.go

```diff
@@ -0,0 +1,56 @@
+package text
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/antonmedv/expr"
+)
+
+type Expression string
+type Output interface{}
+
+type Fixtures map[Expression]Output
+
+var fixtures = Fixtures{
+	`Age(result["started_at"]) > Duration("24h")`: true,
```

Comment:
Can we not make this:

`result["started_at"] > Duration("24h")`

by making BeforeDuration accept an interface{} ?